bin/set-up-account.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
-set -euxo pipefail
+set -euo pipefail
 
 # PROJECT_NAME defaults to name of the current directory.
 # Run this at project root before changing directories
 PROJECT_NAME=$(basename $(PWD))
 
+ACCOUNT=$1
```

Comment:
Should this script error out if ACCOUNT is not provided?

Reply:
I'm not a bash expert but I _think_ it does already:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/447859/196503423-65478549-bf35-49bb-9431-cea61c27ff7f.png">

But then again I'm not sure if there are platform specific differences that I should be aware of

Reply:
oh you're right, works for me as well 🤷